tools/depends/target/ffmpeg/Makefile

```diff
@@ -20,6 +20,7 @@ ffmpg_config += --enable-muxer=asf --enable-muxer=ipod
 ffmpg_config += --enable-encoder=ac3 --enable-encoder=aac
 ffmpg_config += --enable-encoder=wmav2 --enable-protocol=http
 ffmpg_config += --enable-gnutls
+ffmpg_config += --extra-libs="-ldl"
```

Comment:
@lrusak How come this is required on more recent OSes? Curious as I'm wondering if it's needed in the new inputstream addon or is it only required here for libgpg?

Reply:
I didn't look into it too much. It could just be related to linking order, the static libraries, or just the fact that I use fedora.